### PR TITLE
Adding homepage to master

### PR DESCRIPTION
**Pull request by YoseptF**
We finished the 5th project and it was really interesting, we learned a lot about how a website is a design, this was a really insightful project :dancer:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <section class="landing">
+            <h2 style="display: none;">landing section</h2>
```
Comment (pelzolga123) [inline index.html]: Inline styling should be removed.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+            <h2>Get Started With UI Design With These Tips To Speed Up Your Workflow</h2>
+        </article>
+        <article class="four topArticle">
+            <div>
+                <img alt="placeholder image" src="https://via.placeholder.com/75x75/525252"><span>Tomáš Čakloš<em> wrote</em></span>
+            </div>
+            <h2>Get Started With UI Design With These Tips To Speed Up Your Workflow</h2>
+        </article>
+        <section class="callToAction">
+            <h2 style="display: none;">call to action</h2>
```
Comment (pelzolga123) [inline index.html]: Inline styling should be removed.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+                    <li>...plus a seasoned selection of fancy cats.</li>
+                </ul>
+                <span>About Smashing Membership --></span>
+            </div>
+            <img src="https://via.placeholder.com/307x405/2f2f2f" alt="">
+        </section>
+    </section>
+
+    <section class="lastest">
+        <h2 style="display: none;">lastest section</h2>
```
Comment (pelzolga123) [inline index.html]: Inline styling should be removed.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project needs more `article` elements and navbar should position elements as original website.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!

#### Here are the things you need to change before project can be approved:
* [x] Navbar should use div and positions elements as at the source page [Link to page](https://www.smashingmagazine.com/) 
That means, you need to look how it's done on original website and add all missing elements inside your navbar `li` elements (content is not important). The same with `div` element, it should contain `nav` like original website.

* [x] Footer should contain centered links. Right now you have `li` elements, they need to be links. 
* [x] `callToAction` section should be an `article`.
* [x] `features fone` div should be an `article`.
* [x] `episodes ftwo` div should be an `article`.
* [x] `book` section should be an `article`.
* [x] `personOfWeek` div should be an `article`.
* [x] Each `person` div should be an `article`.
* [x] `tableContent` section should be an `article`.
* [x] `newsletter five` section should be an `article`.

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by nikasulo — APPROVED** [review]
# Great job! @YoseptF and @SlowKingV :rocket:.
You may merge your beautiful project :smile: :+1: . See you at the top guys!

**Pull request merged**